.dockerignore

```diff
@@ -1 +1,2 @@
 mysql-data/*
+data/*
```

Comment:
Ah, this would explain why it was taking so long to rebuild docker images, thanks.